src/shapes/group.class.js

```diff
@@ -100,6 +108,22 @@
       }
 
       this.setCoords();
+      this._triggerObjectsEvents();
+    },
+
+    /**
+     * Trigger events on objects.
+     * @private
+     */
+    _triggerObjectsEvents: function () {
+      var _this = this;
+      _this.triggerObjectsEvents.forEach(function (eventName) {
+        _this.on(eventName, function (event) {
+          _this.forEachObject(function (object) {
+            object.fire(eventName, event);
+          });
+        });
+      });
```

Comment:
does this have to be in the library? eventually in my app i can add an events for each group created that does exactly this.

I can wrap my contstructor in a new one that adds this extra functionality:
for each event i m interested in, i can add to each group an event handler that will fire for each object the same event.

```js
MyGroup.on('eventX', function(opts) {
  this.forEachObject(function(object) { object.fire('eventX', opts) });
})
```

Reply:
Yes of course, i just thought it’d be nicer and more consistent to only have to set the event names to listen to directly on the object. Though I realize in terms of consistency that one would expect calling set(‘triggerObjectsEvents’, [´smth’]) to modify the listeners accordingly.